src/content/effective-dart/design.md

```diff
@@ -1200,10 +1200,10 @@ makeGreeting(String who) {
 }
 ```
 
-Note that this guideline only applies to *named* function declarations:
-top-level functions, methods, and local functions. Anonymous function
-expressions infer a return type from their body. In fact, the syntax doesn't
-even allow a return type annotation.
+Note that this guideline only applies to *non-local* function declarations:
+top-level functions, and methods. Local functions and anonymous function
```

Comment:
Maybe "top-level functions, static functions and instance methods".

I don't usually call static functions "methods", although I guess "static method" is commonly used for them. (I think it's a Java way of phrasing it, which began because they don't/didn't have plain functions at all.)